features/cellular/UNITTESTS/at/at_cellularbase/at_cellularbasetest.cpp

```diff
@@ -20,15 +20,13 @@
 
 TEST_GROUP(AT_CellularBase)
```

Comment:
For being consistent with function expansion, please change, void setup() { to:
```
void setup()
{
}
```
as it was originally. Martin will be patching astyle script to ignore expanding functions in the pre-processed objects like TEST_GROUP.

Reply:
Good catch, I'll check these macros and will need to be ignored not to be changed as they are not seen by formatter (is it a function/class/etc) so its treated as function by default.